Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># online-learning-platform\n\n```\nnpm i express express-handlebars mongoose bcrypt connect-mongo express-index\nnpm i node-mon sass --save-dev\n```\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 46941048ce46fe4b2945d39f4070a79dcbc11e32)
+++ b/README.md	(date 1725002059611)
@@ -1,7 +1,7 @@
 # online-learning-platform
 
 ```
-npm i express express-handlebars mongoose bcrypt connect-mongo express-index
+npm i express express-handlebars mongoose bcrypt connect-mongo
 npm i node-mon sass --save-dev
 ```
 
